src/Persistence/Yaml/YamlUtils.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.PowerPlatform.PowerApps.Persistence.Yaml;
+
+public static class YamlUtils
```

Comment:
Please avoid utils pattern

Reply:
I removed the Screens and the EditorState deserialization for now. Let's discuss during standup.